runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerPluginTest.java

```diff
@@ -919,7 +932,8 @@ public void testNewAccountSlayerKill()
 	@Test
 	public void infoboxNotAddedOnLogin()
 	{
-		when(slayerConfig.taskName()).thenReturn(Task.BLOODVELD.getName());
+		when(slayerPlugin.getStoredTaskName()).thenReturn(Task.BLOODVELD.getName());
+		when(slayerPlugin.getStoredAmount()).thenReturn(50);
```

Comment:
Why is the amount set in this test?

Reply:
I figured since this tests [this code branch](https://github.com/runelite/runelite/blob/a58e441f9a236eff8b2012761189f1fe4b6f9db7/runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java#L295) I would set an arbitrary amount value along with the name. If this defaults to 0 otherwise I can remove it.

Reply:
I fiddled around with this some more. The amount check, and `when(slayerConfig.showInfobox()).thenReturn(true);`, which I'll be adding, are both required to properly test this behavior.